index.js

```diff
@@ -79,8 +79,11 @@ module.exports = (number, options) => {
 
 	const exponent = Math.min(Math.floor(options.binary ? Math.log(number) / Math.log(1024) : Math.log10(number) / 3), UNITS.length - 1);
 	// eslint-disable-next-line unicorn/prefer-exponentiation-operator
-	number = Number((number / Math.pow(options.binary ? 1024 : 1000, exponent)).toPrecision(3));
-	const numberString = toLocaleString(number, options.locale);
+	number /= Math.pow(options.binary ? 1024 : 1000, exponent);
+	const digits = options.digits || 3;
```

Comment:
This makes it impossible to set it to `0`. I think handling the defaults should be done higher up where `Object.assign` is.